cloudformation/ad/ad-integration.yaml

```diff
@@ -887,8 +913,12 @@ Outputs:
     Value: !Ref DomainCertificateSecret
   DomainReadOnlyUser:
     Value: !Sub
-      - cn=ReadOnlyUser,ou=Users,ou=${ou},dc=${dc}
-      - { dc: !Join [",dc=", !Split [".", !Ref DomainName ]], ou: !GetAtt Prep.DomainShortName }
+      - cn=ReadOnlyUser,${type}=Users${ou}${dsn},dc=${dc}
```

Comment:
Minor: it's useful to add a comment with the examples of expected values for MicrosoftAD and SimpleAD.